cmd/open.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/CircleCI-Public/circleci-cli/git"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+func projectUrl(remote *git.Remote) string {
+	return fmt.Sprintf("https://app.circleci.com/%s/%s/%s/pipelines",
+		strings.ToLower(string(remote.VcsType)),
+		remote.Organization,
+		remote.Project)
+}
+
+func openProjectInBrowser() error {
+
+	remote, err := git.InferProjectFromGitRemotes()
+
+	if err != nil {
+		return err
```

Comment:
Are these errors human-friendly enough? should we be wrapping here with something nicer?

Reply:
I've upgraded the error reporting:

```
Unable detect which URL should be opened. This command is intended to be run from
a git repository with a remote named 'origin' that is hosted on Github or Bitbucket
Error: This command must be run from inside a git repository
```